gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerFragment.java

```diff
@@ -72,6 +74,26 @@
   private HomeScreenViewModel homeScreenViewModel;
   private MainViewModel mainViewModel;
 
+  @OnClick(R.id.map_type_btn)
```

Comment:
We're trying to move away from Butterknife in favor of Data Binding to ViewModels - could we do that here instead?

Reply:
I agree that the class needs a refactor, but doing that in the same PR would broaden the scope. We can open a new refactor bug to handle that independently. Wdyt?

Reply:
I mainly didn't want to add a new instance of Butterknife binding if we could avoid it, esp. since we already have a MapContainerViewModel. Is it just a question of defining `onMapTypeButtonClick()` in the view model and exposing an `Observable<Event<Nil>>` to trigger the dialog in the view? If so I'd prefer to add it straight away if possible. If you have your hands full atm though please lmk and I can try to send a PR to your branch with the recommended changes!

Reply:
Done